exercises/1.variables/d_type_conversions/d_type_conversions.ml

```diff
@@ -0,0 +1,15 @@
+(* Converting a string to an int *)
+let a = 5 + int_of_string "10"
+
+let x = 3
+(* Convert an int to a string *)
+let s = "There are " ^ x ^ " errors in this exercise."
+
+(* Leave this line as is *)
+let b = 4.0
+(* Convert a float to an int *)
+let c = 1 + b
```

Comment:
Should we add assert statements to this exercise as well? It will be nice to see that "tests" pass.